src/k8s-extension/setup.py

```diff
@@ -33,7 +33,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-VERSION = "1.5.3"
+VERSION = "1.5.4"
```

Comment:
Could you follow the [version guideline](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md) to upgrade the extension version

Reply:
done